weaviate/collections/classes/config_named_vectors.py

```diff
@@ -226,6 +227,46 @@ def text2vec_contextionary(
             vector_index_config=vector_index_config,
         )
 
+    @staticmethod
+    def text2vec_databricks(
+        name: str,
+        *,
+        source_properties: Optional[List[str]] = None,
+        vector_index_config: Optional[_VectorIndexConfigCreate] = None,
+        vectorize_collection_name: bool = True,
+        endpoint: str,
+        instruction: Optional[str] = None,
+    ) -> _NamedVectorConfigCreate:
+        """Create a named vector using the `text2vec-mistral` model.
+
+        See the [documentation](https://weaviate.io/developers/weaviate/modules/retriever-vectorizer-modules/text2vec-mistral)
+        for detailed usage.
+
+        Arguments:
+            `name`
+                The name of the named vector.
+            `source_properties`
+                Which properties should be included when vectorizing. By default all text properties are included.
+            `vector_index_config`
+                The configuration for Weaviate's vector index. Use wvc.config.Configure.VectorIndex to create a vector index configuration. None by default
+            `endpoint`
+                The endpoint to use.
+            `instruction`
+                The instruction strategy to use. Defaults to `None`, which uses the server-defined default.
+            `vectorize_collection_name`
+                Whether to vectorize the collection name. Defaults to `True`.
```

Comment:
Nitpicking, but could this match the order of the arguments in the function signature?